server_addon/houdini/server/settings/create.py

```diff
@@ -26,7 +26,7 @@ class CreateStaticMeshModel(BaseSettingsModel):
         default_factory=list,
         title="Default Products"
     )
-    static_mesh_prefixes: str = Field("S", title="Static Mesh Prefix")
+    static_mesh_prefix: str = Field("S", title="Static Mesh Prefix")
```

Comment:
Out of curiosity - with this renamed; what does [this](https://github.com/ynput/OpenPype/blob/4bafde264199f8c155e1e6c8eba920de07a2a938/openpype/settings/ayon_settings.py#L503) intend to do?

Reply:
yeah, I spotted that earlier and I'm curios too to be honest.
Also, running these commands in Houdini python shell or Maya console amazes me! 

```python
from openpype.settings import get_current_project_settings
project_settings = get_current_project_settings()
print("maya", project_settings["maya"]["create"]["CreateUnrealStaticMesh"])
print("houdini", project_settings["houdini"]["create"]["CreateStaticMesh"])
```
and here are the results
```python
maya {'enabled': True, 'default_variants': ['', '_Main'], 'collision_prefixes': ['UBX', 'UCP', 'USP', 'UCX'], 'static_mesh_prefix': 'S'}
houdini {'enabled': True, 'default_variants': ['Main'], 'static_mesh_prefix': 'S', 'collision_prefixes': ['UBX', 'UCP', 'USP', 'UCX']}
```
I can't understand why it happens. 

---

but the thing is 
in Ayon settings, we should use the same key in the model class definition and its default values 
however in class definition 
we uses this key 
```
static_mesh_prefixes: str = Field("S", title="Static Mesh Prefix")
```
but in default values we uses another key
```
"static_mesh_prefix": "S",
```
also, in the corresponding Openpype settings, I can't find `static_mesh_prefixes` at all! 

So, I don't know maybe the code your mentioning above is for a backward compatibility issue ? IDK!

Reply:
Silly me I'm printing the values after fixing the issue :) 
Here are how it looked like before this PR.
> So, I needed only to switch to an older bundle, This is way cooler! 

```python
from openpype.settings import get_current_project_settings
project_settings = get_current_project_settings()
print("maya", project_settings["maya"]["create"]["CreateUnrealStaticMesh"])
print("houdini", project_settings["houdini"]["create"]["CreateStaticMesh"])
```

Reply:
Thanks - no idea myself either; this is a question for someone else I suppose.